CMakeLists.txt

```diff
@@ -276,6 +280,21 @@ if(NOT LLVM_ENABLE_RTTI AND NOT PHASAR_IN_TREE)
   message(FATAL_ERROR "PhASAR requires a LLVM version that is built with RTTI")
 endif()
 
+# Z3 Solver
+if(PHASAR_USE_Z3)
+  # This z3-version is the same version LLVM requires; however, we cannot just use Z3 via the LLVM interface
+  # as it lacks some functionality (such as z3::expr::simplify()) that we require
+  find_package(Z3 4.7.1 REQUIRED)
```

Comment:
Have looked it up: [find_package](https://cmake.org/cmake/help/latest/command/find_package.html#basic-signature) allows to specify a minimum version or a version range. In case of a single version, you can provide the option `EXACT` to request an exact match instead.
So, this find_package invocation looks correct